esrally/driver/runner.py

```diff
@@ -332,7 +337,8 @@ async def __call__(self, es, params):
         It expects a parameter dict with the following mandatory keys:
 
         * ``body``: containing all documents for the current bulk request.
-        * ``bulk-size``: the number of documents in this bulk.
+        * ``bulk-size``: An indication of the bulk size denoted in ``unit``.
```

Comment:
That's tricky because we cannot support `bulk-size` without introducing bottlenecks when the unit is not "docs" (because we need to count the bulk items ourselves which requires deserializing the request or the response which we now avoid at all costs). As I've argued above I feel this is special enough that we can just remove it.

Reply:
Discussed with @danielmitterdorfer